scripts/mkcloud

```diff
@@ -974,6 +983,7 @@ Steps:
     setup_aliases:  set aliases for the nodes (usually needed for batch step)
     batch:          build proposals from exported crowbar_batch YAML file
     setuplonelynodes: boot a number (defined by nodenumberlonelynode) of non-crowbar registered nodes in the admin network
+    setupironicnodes: boot a number of empty nodes to be discovered by the ironic service
```

Comment:
Currently there's no ironic discovery integrated. I'd change this to "to be used".